src/content/blog/ddev-bundled-tools-using-custom-versions.md

```diff
@@ -0,0 +1,52 @@
+
+---
+title: "How-to downgrade terminus in DDEV and customize bundled tools"
+pubDate: 2025-06-12
+summary: How to upgrade/downgrade a utility provided by DDEV, or add a custom one for a given project
+author: Bill Seremetis
+featureImage:
+  src: /img/blog/2025/06/ddev-tool-install.png
+  alt: "Installing custom software packages in the containerized environment"
+  shadow: true
+categories:
+  - Guides
+  - DevOps
+---
+
+_This guest post is by DDEV community member and [Drupal](https://drupal.org)
+contributor [Bill Seremetis](/blog/author/bill-seremetis/) and sponsored by
+[Annertech](https://www.annertech.com)._
+
+DDEV comes bundled with a predefined set of tools, `terminus` being one of them.
+Latest releases of `terminus` are not compatible with older PHP versions, 
+thus there is a need to downgrade the package inside DDEV.
+
+This guide covers how and will also explain how to use the same technique to install
+custom tools too.
+
```

Comment:
Intro is doing great, thanks for the fixes. 

Please add a paragraph here which lists some of the things you can do, linking to the docs. 

In your case, you used a Dockerfile.*, please also mention the `webimage_extra_packages` approach.